### PR TITLE
kubernetes: Implement Adjust Service dialog

### PR DESCRIPTION
**Pull request by stefwalter**
Only changing the number of replicas works for now.

### CODE REVIEW

**Comment by stefwalter** [comment]
Depends on #2249, #2250, #2237

**Comment by stefwalter** [comment]
Screenshot. Although I've renamed the 'Deploy' button to match designs:

screenshot from 2015-05-06 04-03-24

**Comment by stefwalter** [comment]
Rebased.

File: pkg/kubernetes/adjust.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+            var meta = service.metadata || { };
+            var spec = service.spec || { };
+
+            /* We can't edit the name for now */
+            $("#adjust-name")
+                .val(meta.name)
+                .attr("disabled", "disabled");
+
+            /* Generate table rows for each replication controller this service matches */
+            var rcs = kube.select(spec.selector || { }, meta.namespace, "ReplicationController");
```
Comment (stefwalter) [inline pkg/kubernetes/adjust.js]: Instead of selecting replication controllers by label, we must select them by the labels in their template.

**Comment by stefwalter** [comment]
Depends on #2245

**Comment by stefwalter** [comment]
Testing information:

 * Click the check box, then click edit next to a service, type a new number of replicas, Click 'Adjust'
 * Watch the number of containers change for the service.

**Comment by mvollmer** [comment]
The table columns jump when the edit buttons appear/disappear.

File: pkg/kubernetes/cluster.html [hunk 2]
```diff
@@ -220,14 +223,54 @@ <h4 class="modal-title" translatable="yes">Deploy Application</h4>
...
   <script>
       require([
           "base1/cockpit",
           "translated!base1/po",
           "kubernetes/angular",
           "kubernetes/deploy",
+          "kubernetes/adjust",
           "kubernetes/app",
-          "kubernetes/dashboard"
+          "kubernetes/dashboard",
```
Comment (mvollmer) [inline pkg/kubernetes/cluster.html]: Accidental change?

**Comment by stefwalter** [comment]
> The table columns jump when the edit buttons appear/disappear.

This was fixed in a later branch, but moved it here.

File: pkg/kubernetes/cluster.html (hunk 2 above)
Comment (stefwalter) [inline pkg/kubernetes/cluster.html]: Removed.

**Pull request closed without merging**